samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -505,6 +505,10 @@ public String getCurrentTimeElapse() {
     return timeElapse;
   }
 
+  public String getLocale() {
+    return PortalUtils.getLocaleString(new ResourceLoader().getLocale());
```

Comment:
Optional improvement:
this is an old pattern for getting the user locale it would be ideal to use the preferences service to get the users locale org.sakaiproject.user.api.PreferencesService#getLocale